CHANGELOG.rst

```diff
@@ -7,6 +7,12 @@ foursight-smaht
 Change Log
 ----------
 
+0.1.0
+=====
+* Update to Python 3.11.
```

Comment:
It's a good idea to do a blank line above a bullet list. Some tools don't recognize it if it's pushed up against another paragraph.

Reply:
Fixed